src/lightly_train/_task_models/dinov2_ltdetr_object_detection/transforms.py

```diff
@@ -141,7 +141,7 @@ def resolve_auto(self, model_init_args: dict[str, Any]) -> None:
         if self.normalize == "auto":
             normalize = model_init_args.get("image_normalize")
             if normalize is None:
-                self.normalize = None
+                self.normalize = NormalizeArgs()
```

Comment:
I think we have to make a finer-grained check here. There are 3 scenarios:
1. `model_init_args` is empty or "image_normalize" is not set
2. `model_init_args` is not empty and "image_normalize" is set to `None`
3. `model_init_args` is not empty and "image_normalize" is a dict

And we would want the following outcome:
1. Default to `NormalizeArgs()` or `None` --> You decide what makes more sense
2. `normalize = None`
3. `normalize = NormalizeArgs.from_dict(...)`